packages/rancher-monitoring/generated-changes/patch/templates/_helpers.tpl.patch

```diff
@@ -57,6 +57,15 @@
 +{{- end -}}
 +{{- end }}
 +
++{{- define "kubelet.serviceMonitor.resourcePath" -}}
++{{- $kubeTargetVersion := default .Capabilities.KubeVersion.GitVersion .Values.kubeTargetVersionOverride }}
++{{- if and (semverCompare ">=1.20.0-0" $kubeTargetVersion) (eq .Values.kubelet.serviceMonitor.resourcePath "/metrics/resource/v1alpha1") -}}
```

Comment:
This feels kinda weird, users can never set it to this path in future. Perhaps thats ok if it's really gone from there.

Reply:
This path was deprecated in an earlier k8s version and removed entirely in k8s 1.20, so it should be gone for good.

Reply:
Ok. Well then its just making sure its inline with semver compare convo from other PR.